bobtemplates/odoo/hooks.py

```diff
@@ -151,9 +154,29 @@ def post_render_model(configurator):
 # addon hooks
 #
 
+def _set_oca_website(configurator):
+    website = configurator.variables['copyright.website']
+    website = OCA_WEBSITE
+    try:
+        result = subprocess.check_output(
+            "git config --get remote.origin.url",
+            shell=True,
+            stderr=subprocess.STDOUT,
+        )
+        if result:
+            regex = "[^\/]+(?=\.git$)"
+            match = re.search(regex, result)
+            if match:
+                website = website.replace('<repo>', match.group(0))
+    except:
```

Comment:
`except Exception`, but why catching exceptions here?

Reply:
to put default website